paywall/src/stories/interface/StaticDemo.stories.js

```diff
@@ -31,7 +31,7 @@ const store = createUnlockStore({
   },
   router: {
     location: {
-      pathname: '/demo/' + myLock.address,
+      pathname: '/newdemo?lock=' + myLock.address,
```

Comment:
Ideally I'd love to use the same kind of pattern that we use for the donation page here with the config encoded as a query string? or maybe just turn the json into a valid query string?